src/main/java/org/jbehavesupport/engine/reporter/StepLoggingReporter.java

```diff
@@ -57,6 +58,10 @@ public class StepLoggingReporter extends AbstractLoggingReporter {
     private Deque<TestDescriptor> currentStepDescriptor = new ArrayDeque<>();
 
     private boolean isInBeforeStories = false;
+
+    private boolean isInBeforeScenario = false;
+
+    private boolean isInAfterScenario = false;
```

Comment:
ultra nitpick but I would remove the new lines on here 😄 
```suggestion
    private boolean isInBeforeStories = false;
    private boolean isInBeforeScenario = false;
    private boolean isInAfterScenario = false;
```